sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -57,9 +57,17 @@ class Mean(object):
 
   class Globally(ptransform.PTransform):
     """combiners.Mean.Globally computes the arithmetic mean of the elements."""
+    def __init__(self, label=None, as_view=False, has_defaults=True):
```

Comment:
<!--new_thread; commit:692e00a064e38575fe4c49e9c7ac209fc38a9c6a; resolved:0-->
We spent quite a bit of time in the past to remove label arguments in favor of the `>>` syntax. I do not think we should re-introduce them.